config/default-amo.js

```diff
@@ -22,7 +22,6 @@ module.exports = {
         'data:',
         amoProdCDN,
         staticHost,
-        'https://www.google-analytics.com',
```

Comment:
On Line 17, there is the same URL suffixed with the js file. I guess we could reuse `analyticsHost` for this value too.